pkg/logentry/stages/labeldrop.go

```diff
@@ -0,0 +1,61 @@
+package stages
+
+import (
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+	"github.com/prometheus/common/model"
+)
+
+const (
+	// ErrEmptyLabelDropStageConfig error returned if config is empty
+	ErrEmptyLabelDropStageConfig = "drop_label stage config cannot be empty"
```

Comment:
drop_label and labeldrop ? There is some inconsistency here. Should we rename this to `drop_labels` everywhere ? WDYT ? naming is hard but also important.

Reply:
thanks for finding that, I switched that out to labeldrop, I picked "labeldrop" over "drop_label" because the relabel action to drop labels is called labeldrop so thought it would be more consistent.